06-reproj.Rmd

```diff
@@ -167,12 +167,13 @@ st_distance(london2, london_proj)
 ## Which CRS to use?
 
 The question of *which CRS* is tricky, and there is rarely a 'right' answer:
-"There exist no all-purpose projections, all involve distortion when far from the center of the specified frame" [@bivand_applied_2013].
-For geographic CRSs, the answer is often [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#A_new_World_Geodetic_System:_WGS_84), not only for web mapping (covered in the previous paragraph) but also because GPS datasets and thousands of raster and vector datasets are provided in this CRS by default.
+"There exist no all-purpose projections, all involve distortion when far from the center of the specified frame" [@bivand_applied_2013**.
```

Comment:
Missing closing square bracket.

Reply:
Ya, indeed. Sorry. I use Emacs and because I normally don't use markdown, my markdown-mode settings are not fine-tuned very nicely and I have some weird auto-formatting going on. I noticed one earlier and force-pushed a commit to correct it but I missed this one. I am glad you spotted it!